webui/react/src/components/LogViewer/LogViewerFilters.tsx

```diff
@@ -89,8 +89,8 @@ const LogViewerFilters: React.FC<Props> = ({ onChange, onReset, options, values
             itemName={LABELS.allocationIds}
             value={values.allocationIds}
             onChange={handleChange('allocationIds', String)}>
-            {selectOptions?.allocationIds?.map(id => (
-              <Option key={id} value={id}>{id || 'No Allocation ID'}</Option>
+            {selectOptions?.allocationIds?.map((id, index) => (
+              <Option key={id || `no-id-${index}`} value={id}>{id || 'No Allocation ID'}</Option>
```

Comment:
extra precaution of if there are multiple options that have id of empty strings. This shouldn't happen from the API side, but you never know.

Reply:
we could probably always go with index. it'd be less stable rendering wise maybe but should do the job.since it's for testing. works as is too

Reply:
I was bit hesitant to just use the `index` in the case where new ids came in (since the log filter options may change over time while a trial is running for instance) and with the index as the key, the options might not re-render properly. Not 100% sure about that.